book/src/development/method_checking.md

```diff
@@ -23,7 +23,7 @@ impl<'tcx> LateLintPass<'tcx> for OurFancyMethodLint {
         // Check our expr is calling a method with pattern matching
         if let hir::ExprKind::MethodCall(path, _, [self_arg, ..]) = &expr.kind
             // Check if the name of this method is `our_fancy_method`
-            && path.ident.name == sym!(our_fancy_method)
+            && path.ident.name.as_str() == "our_fancy_method"
```

Comment:
There's a paragraph under this codeblock that doesn't make much sense anymore with this changed

> Additionally, we use the [clippy_utils::sym!](https://doc.rust-lang.org/stable/nightly-rustc/clippy_utils/macro.sym.html) macro to conveniently convert an input our_fancy_method into a Symbol and compare that symbol to the [Ident](https://doc.rust-lang.org/beta/nightly-rustc/rustc_span/symbol/struct.Ident.html)'s name in the [PathSegment](https://doc.rust-lang.org/beta/nightly-rustc/rustc_hir/hir/struct.PathSegment.html) in the [MethodCall](https://doc.rust-lang.org/beta/nightly-rustc/rustc_hir/hir/enum.ExprKind.html#variant.MethodCall).

We can probably just remove that, I don't think the .as_str() comparison needs an explanation

Reply:
Done, just removed it.